td-agent/Rakefile

```diff
@@ -326,6 +326,7 @@ class BuildTask
           "etc/td-agent/td-agent.conf",
           "etc/td-agent/td-agent.conf.tmpl",
           "etc/logrotate.d/td-agent",
+          "usr/lib/tmpfiles.d/td-agent.conf",
```

Comment:
CentOS 6 is not systemd based. This should be needed for CentOS 7 and CentOS 8. Not for CentOS 6.

Reply:
NOTE: systemd-tmpfiles is the successor systemd module of tmpwatch.
https://www.itmedia.co.jp/help/tips/linux/l0609.html

Reply:
Hmm..., tmpwatch is a bit messy for modifying installation phase....